homeassistant/components/weatherflow/strings.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      }
+    },
+    "error": {
+      "address_in_use": "The host address is already in use. Please specify a different host address below.",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "temperature": {
+        "name": "Temperature"
+      },
+      "air_density": {
+        "name": "Air density"
+      },
+      "dew_point": {
+        "name": "Dew point"
+      },
+      "battery_voltage": {
+        "name": "Battery voltage"
+      },
+      "feels_like": {
+        "name": "Feels like"
+      },
+      "illuminance": {
+        "name": "Illuminance"
+      },
+      "lightning_average_distance": {
+        "name": "Lightning average distance"
+      },
+      "lightning_count": {
+        "name": "Lightning count"
+      },
+      "precipitation_type": {
+        "name": "Precipitation type"
+      },
+      "rain_amount": {
+        "name": "Rain amount"
+      },
+      "rain_rate": {
+        "name": "Rain rate"
+      },
+      "humidity": {
+        "name": "Humidity"
+      },
+      "rssi": {
+        "name": "Rssi"
```

Comment:
I would go for RSSI or remove it and use the device class translation "Signal strength"

Reply:
So... if i use signal strength I don't need a translation key because device-classes have them built in... 👍️ (will do)

Reply:
Exactly

Reply:
Nixed it!